### PR TITLE
Update zh_cn.lang

### PR DESCRIPTION
**Pull request by DYColdWind**

### CODE REVIEW

File: src/main/resources/assets/techreborn/lang/zh_cn.lang [hunk 1]
```diff
@@ -375,6 +375,7 @@ item.techreborn.dustsmall.diorite.name=小堆闪长岩粉
 item.techreborn.dustsmall.granite.name=小堆花岗岩粉
 
 item.techreborn.cell.name=空单元
+item.techreborn.cell.fluid.name=$流体$单元
```
Comment (modmuss50) [inline src/main/resources/assets/techreborn/lang/zh_cn.lang]: This wont work, you will want to have the following:

`item.techreborn.cell.fluid.name=$fluid$单元`

**Review by modmuss50 — CHANGES_REQUESTED** [review]
Almost correct, but this wont work as it only replaces `$fluid$` with the fluid name.

File: src/main/resources/assets/techreborn/lang/zh_cn.lang (hunk 1 above)
Comment (DYColdWind) [reply src/main/resources/assets/techreborn/lang/zh_cn.lang]: Thank you for pointing out the mistake ...

Comment (liach) [reply src/main/resources/assets/techreborn/lang/zh_cn.lang]: Looks funky! modmuss, you should use upper case characters to better indicate such tokens.

**Review by modmuss50 — APPROVED** [review]

**Pull request merged**